forge/src/main/java/org/openjdk/skara/forge/github/GitHubPullRequest.java

```diff
@@ -736,7 +736,9 @@ public URI headUrl() {
 
     @Override
     public Diff diff() {
-        var files = request.get("pulls/" + json.get("number").toString() + "/files").execute();
+        var files = request.get("pulls/" + json.get("number").toString() + "/files")
+                           .param("per_page", "50")
```

Comment:
Can we add a comment explaining the need for per_page? Something like:

"Need to specify an explicit per_page < 70 to guarantee that we get patch information in the result set."

Reply:
Added.